dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/organisationunit/OrganisationUnitLocationController.java

```diff
@@ -72,12 +71,12 @@
      * Get Organisation Units within a distance from a location
      */
     @GetMapping( value = "/withinRange", produces = { "*/*", APPLICATION_JSON_VALUE } )
-    public void getEntitiesWithinRange(
+    @ResponseBody
+    public List<OrganisationUnit> getEntitiesWithinRange(
```

Comment:
This cannot be tested with spring MVC as it is using `within` SQL function. Please look more careful here.